newIDE/app/src/UI/Theme/DarkTheme/Mosaic.css

```diff
@@ -23,3 +23,20 @@
 .mosaic-gd-dark-theme.mosaic.mosaic-blueprint-theme .mosaic-preview .mosaic-window-toolbar.draggable:hover {
   background: #3c4698 !important;
 }
+
+/* Separator draggable background */
+.mosaic-gd-dark-theme .mosaic-split.-column .mosaic-split-line {
+  height: 2px;
+  border-bottom: 1px solid #888888 !important;
+  border-top: 1px solid #888888 !important;
+  width: 8%;
+  margin: auto;
+  background-color: #444444;
+  bottom: auto !important;
```

Comment:
Are you sure this is necessary?

Reply:
margin: auto; for center separator.
background-color: #444444; for better look on separator.
bottom: auto !important; for override because otherwise separator is not align vertically with the draggable zone.